metsim/physics.py

```diff
@@ -161,84 +162,85 @@ def svp_slope(temp, a=0.61078, b=17.269, c=237.3):
     return (b * c) / ((c + temp) * (c + temp)) * svp(temp, a=a, b=b, c=c)
 
 
-def solar_geom(elev: float, lat: float) -> dict:
+
+
+@jit(nopython=True, cache=True)
+def solar_geom(elev, lat):
     """
     Flat earth assumption
     """
-    dayperyear = int(np.ceil(consts['DAYS_PER_YEAR']))
+    # optical airmass by degrees
+    OPTAM = [2.90,  3.05,  3.21,  3.39,  3.69,  3.82,  4.07, 
```

Comment:
There is not one in `mtclim_vic.c` in the VIC code surrounding the definition, but maybe within other documentation somewhere.